use/StartSNP.Rmd

```diff
@@ -97,18 +97,18 @@ We observed that heterozygosity varies among loci.
 We observed no difference between expected and observed heterozygosity.
 
 
-Basic statistics with hierfstat. Populations are states. The function `basicstat`
+Basic statistics with *hierfstat*. Populations are states. The function `basic.stats()`
 provides the observed heterozygosity ($H_o$), mean gene diversities within
-population ($H_s$), $F_{is}$, and $F_{st}$ The function `boot.ppfis` provides
-confidence interval for $F_{is}$ The fuction `indpca`  does an PCA on the
-centered matrix of individual's allele frequencies.
+population ($H_s$), $F_{is}$, and $F_{st}$ The function `boot.ppfis()` provides
```

Comment:
@smanel, since the pull request is from me, it's my responsibility to add the full stops. It will be done momentarily.